apps/frontend/src/app/5_pages/ErrorPage/ErrorPage.tsx

```diff
@@ -24,16 +25,18 @@ export const ErrorPage: FC = () => {
         <title>{t(translations.errorPage.meta.title)}</title>
       </Helmet>
       <Header />
-      <div className="container flex flex-col flex-grow items-center justify-center">
-        <h1>{t(translations.errorPage.title)}</h1>
-        <p>{t(translations.errorPage.description)}</p>
-        <p>
-          <i>{error.statusText || error.message}</i>
-        </p>
-        {process.env.NODE_ENV === 'development' && (
-          <pre>{JSON.stringify(error, undefined, 2)}</pre>
-        )}
-      </div>
+      <ErrorRenderer
+        error={
+          <>
+            <p className="w-full p-3">
+              <i>{error.statusText || error.message}</i>
+            </p>
+            {process.env.NODE_ENV === 'development' && (
+              <pre>{JSON.stringify(error, undefined, 2)}</pre>
+            )}
```

Comment:
Since we're editing this part now let's also update it to use Paragraph component